eng/targets/Settings.props

```diff
@@ -17,6 +17,7 @@
     <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
     <ToolsetPackagesDir>$(RepoRoot)build\ToolsetPackages\</ToolsetPackagesDir>
     <RoslynPortableTargetFrameworks>netcoreapp3.1;net472</RoslynPortableTargetFrameworks>
+    <RoslynLibraryTargetFrameworks>netcoreapp3.1;netstandard2.0</RoslynLibraryTargetFrameworks>
```

Comment:
Why did you multi-target between `netcoreapp3.1` and `netstandard2.0` here? Yes this will fix the scenario where you build for `netcoreapp3.1` but it will not fix when you build for `net472`. The latter will still have all the same problems that we have today. Why not instead multi-target between `netcoreapp3.1` and `net472`? That way both desktop and core benefit from this change.